beacon_node/beacon_chain/src/kzg_utils.rs

```diff
@@ -27,11 +27,11 @@ pub fn validate_blob<T: EthSpec>(
 pub fn validate_blobs<T: EthSpec>(
     kzg: &Kzg<T::Kzg>,
     expected_kzg_commitments: &[KzgCommitment],
-    blobs: &[Blob<T>],
+    blobs: Vec<&Blob<T>>,
     kzg_proofs: &[KzgProof],
 ) -> Result<bool, KzgError> {
     let blobs = blobs
-        .iter()
+        .into_iter()
         .map(|blob| ssz_blob_to_crypto_blob::<T>(blob)) // Avoid this clone
```

Comment:
We avoided the clone so can remove this comment too.

Reply:
Fixed in https://github.com/sigp/lighthouse/pull/4852/commits/b20bd420d7e62da25f54a6e8e3b4b1311e6f5f4e